lib/send_text.rb

```diff
@@ -0,0 +1,25 @@
+require 'twilio-ruby'
+require 'dotenv/load'
+
+class SendText
+    def send
+        account_sid = ENV['TWILIO_ACCOUNT_SID']
+        auth_token = ENV['TWILIO_AUTH_TOKEN']
+        @time = Time.now + (1 * 60 * 60)
+
+        @client = Twilio::REST::Client.new(account_sid, auth_token)
```

Comment:
You are creating a new Client every time you send a message. Could you instead create this client once and use the same one to send multiple messages?